posthog/settings/__init__.py

```diff
@@ -77,8 +77,11 @@
 ACTION_EVENT_MAPPING_INTERVAL_SECONDS = get_from_env("ACTION_EVENT_MAPPING_INTERVAL_SECONDS", 300, type_cast=int)
 
 ASYNC_EVENT_PROPERTY_USAGE = get_from_env("ASYNC_EVENT_PROPERTY_USAGE", True, type_cast=str_to_bool)
-EVENT_PROPERTY_USAGE_INTERVAL_SECONDS = get_from_env(
-    "ASYNC_EVENT_PROPERTY_USAGE_INTERVAL_SECONDS", 86400, type_cast=int
+EVENT_PROPERTY_USAGE_INTERVAL_CRON = get_from_env(
+    #  run frequently in dev but once a day by default
+    "ASYNC_EVENT_PROPERTY_USAGE_INTERVAL_CRON",
```

Comment:
not a big fan of this.

It's an env var, hence easily configurable, so I'd propose if someone is doing things around property usage, to do 
`ASYNC_EVENT_PROPERTY_USAGE_INTERVAL_CRON="* * * * *" ./bin/start`

vs setting this here like so 😬

Reply:
... unless there's something that breaks in DEV without this config (which is the common thing across settings where we explicitly set a default for DEV)

Reply:
Ah, not a fan of the X if DEBUG else Y

No strong opinion here :)

Reply:
Even my line comments are off by one, sorry 😂